invenio_rest/decorators.py

```diff
@@ -34,7 +34,12 @@
 
 
 def require_content_types(*allowed_content_types):
-    """Decorator to test if proper content-type is provided."""
+    """Decorator to test if proper content-type is provided.
```

Comment:
- 'content type' or ``Content-Type``?

(Everywhere else we refer to it as 'content type' or 'content types'.)